Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -2335,6 +2337,11 @@ Color RenderStyle::colorByApplyingColorFilter(const Color& color) const
     return transformedColor;
 }
 
+Color RenderStyle::resolvedColorByApplyingColorFilter(const StyleColor& color) const
```

Comment:
this name is a bit confusing, it suggests that we're resolving the color by using the color filter, rather than, resolving the color then applying the color filter.

Maybe `resolvedColorWithColorFilter` would be more accurate or what Antti originally suggested. Also we're using `resolve` vs. `resolved` in places for colors, would be nice to be consistent.